backend/src/govsg/routes/index.ts

```diff
@@ -1,2 +1,3 @@
 export { default as govsgCampaignRoutes } from './govsg-campaign.routes'
 export { default as govsgTemplateRoutes } from './govsg-template.routes'
+export * from './govsg-transactional.routes'
```

Comment:
not a fan of default exports, but I see that it's being used inconsistently in other parts of the codebase too